packages/react-dom/src/client/ReactDOM.js

```diff
@@ -1268,7 +1268,16 @@ const ReactDOM: Object = {
 
   // Temporary alias since we already shipped React 16 RC with it.
   // TODO: remove in React 17.
-  unstable_createPortal: createPortal,
+  unstable_createPortal(...args) {
+    lowPriorityWarning(
```

Comment:
We should make sure to show this warning once at most. Please deduplicate it.

Reply:
I used `hasLoggedCreatePortalDeprecationWarning` to check if already logged. Variable name too verbose?